spec/ci/php70_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe 'PHP 7.0 CI' do
+  include_context 'ci' do
+    let(:soe_packages) { [
+      'socat',
+      'ssmtp'
+    ] }
+
+    let(:php_version) { '7.0' }
+
+    let(:php_packages) { [
```

Comment:
Can you pull this package list out into a constant and reference it here?

See `Constants::PHP54_PACKAGES` in `spec/_shared/constants.rb` and `spec/soe/php54_spec.rb` for an example.